core/lowering/passes/module_fallback.cpp

```diff
@@ -62,7 +62,9 @@ void NotateModuleForFallback(
   }
 
   for (const auto sub_mod : mod.named_children()) {
-    NotateModuleForFallback(sub_mod.value, sub_mod.name, method_name, forced_fallback_modules);
+    if (sub_mod.value.find_method(method_name)) {
```

Comment:
Is there ever a case where we might pass in a name other than "forward" as the method name here? In that case, would the desired behavior be to only recurse through submodules with that specific method (and ignore others, and their children)?

Reply:
What we really should be doing is finding the Method calls and using that to recurse vs all children probably